extensions/blocks/podcast-player/templates/podcast-header-title.php

```diff
@@ -33,7 +33,8 @@ class="jetpack-podcast-player__current-track-title <?php echo esc_attr( $primary
 		echo esc_html( $track['title'] );
 		if ( ! empty( $track_link ) ) :
 			// Prevent whitespace between title and link to cause a jump when JS kicks in.
-			?><a
+			?>
+			<a
```

Comment:
This might be problematic. See conversation here:
https://github.com/Automattic/jetpack/pull/16054#discussion_r436007463

Reply:
Cool. Thanks for that. I've set to disable the rule for that section of code in e0920df60 to avoid a future regression for this reason. cc: @obenland